azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKEventHandler.kt

```diff
@@ -340,9 +354,8 @@ internal class CallingSDKEventHandler(
 
         val callState = call?.state
         var callEndStatus = Pair(0, 0)
-
         when (callState) {
-            CallState.CONNECTED -> {
+            CallState.CONNECTING, CallState.CONNECTED -> {
```

Comment:
Will it trigger duplicate `addParticipants(call!!.remoteParticipants)` for non 1 to 1 calls?

Reply:
No Pavel, participant add is by mri check. We need it as for 1 to 1 calling participants are remote in connecting state.